packages/vitest/src/typecheck/collect.ts

```diff
@@ -26,7 +26,7 @@ interface LocalCallDefinition {
   end: number
   name: string
   type: 'suite' | 'test'
-  mode: 'run' | 'skip' | 'only' | 'todo'
+  mode: 'run' | 'skip' | 'only' | 'todo' | 'repeats'
```

Comment:
Okay, will try to change this.

Reply:
Without this new mode it's alot harder. If there is a `repeats` option in a `describe` and no options in a `test`, the `repeats` option in the `test` will be overlapped by the one in the `describe`. With this new mode we can just look at it instead of looking for a `repeats` option so this error doesn't happen.

Reply:
No, it introduces a lot of unnecessary "ifs". We should only look at `repeats` option, and remove "repeats" mode altogether.